File: Server/SD_Server.Domain/Features/Professionals/Professional.cs
using SD_Server.Domain.Base;
using SD_Server.Domain.Enum;

namespace SD_Server.Domain.Features.Professionals
{
    public class Professional : BaseEntity
    {
        public string Name { get; set; } 
        public string Email { get; set; } 
        public string? Phone { get; set; }
        public string Cref { get; set; } 
        public string? Bio { get; set; } 
        public string PasswordHash { get; set; } 
        public StatusEnum Status { get; set; }
        public string? Specialty { get; set; }
        public string? Methodology { get; set; }
        public decimal? Price { get; set; }
        public string? Experience { get; set; }
    }
}

File: Mobile/app/lib/core/features/profissionais/domain/entities/professional.dart
class Professional {
  final int id;
  final String name;
  final String email;
  final String? phone;
  final String cref;
  final String? bio;
  final int status;
  final String? specialty;
  final String? methodology;
  final double? price;
  final String? experience;

  Professional({
    required this.id,
    required this.name,
    required this.email,
    this.phone,
    required this.cref,
    this.bio,
    required this.status,
    this.specialty,
    this.methodology,
    this.price,
    this.experience,
  });
}
